example_chain/go.mod

```diff
@@ -245,7 +245,7 @@ require (
 )
 
 // LOCAL REPLACEMENTS FOR Cosmos EVM
-replace github.com/cosmos/evm => ..
+//replace github.com/cosmos/evm => ..
```

Comment:
I think we want this replace still. Why should we remove it?

Reply:
It seems to break those that import this repo as a module

Reply:
fixed